### ISSUE TITLE
Major issue on 1.0.0 and 1.1.0

### ISSUE THREAD

**Issue by mathgirl718** [MelonSpeedruns/TotkRandomizer#23]
So, there's a bug from 1.0.0/1.1.0, however it is completely consistent on rando, while having been unable to replicate on vanilla, it skips all intro stuff (which would be nice to be added) but then when you go through all the steps to open the door of time and get off the great sky island you can't unless you're on version 1.1.1 or higher. Got a full [video](https://youtu.be/fKoU7Cetfro) of me doing so. Also gives you a free paraglider and all the abilities on 1.0.0 (presumably 1.1.0 as well). Actually seems to be a wide variety of issues on 1.0.0, probably 1.1.0, lots of cutscenes cut out and other things on GSI.

**Comment by MelonSpeedruns** [comment]
yeah I fixed all those issues in the latest commits, try building it yourself with Visual Studio? I don't have the time to work on this project anymore. Maybe I'll go back to it one day.

**Comment by darkzeroat** [comment]
yep the door doesnt open on 1.1.0, no matter what you do (i also built the newest version from github)
tried fixing this for a few days now but updating to 1.1.1 or higher fixed it (only tried 1.1.1 and 1.2.1 tho)
1.1.1 does include an official fix for the door not opening since that seems to be a bug on lower versions so i guess that has to do with it
